sqldelight-gradle/plugin/src/main/kotlin/com/squareup/sqldelight/gradle/VerifyMigrationTask.kt

```diff
@@ -106,5 +114,10 @@ open class VerifyMigrationTask : SourceTask() {
     inclusion().exclude().propertyName("importedForeignKeys")
     inclusion().exclude().propertyName("deferrable")
     inclusion().exclude().propertyName("initiallyDeferred")
+    // Definition changes aren't important, its just things like comments or whitespace.
+    inclusion().exclude().propertyName("definition")
```

Comment:
Are you forced to call `inclusion().exclude()` every time? Or could it all be `.apply`'d to a single call?

Reply:
apply should work!